tests/test_index.py

```diff
@@ -1,1198 +0,0 @@
-# Copyright (C) 2014 Anaconda, Inc
```

Comment:
10% less test time

Reply:
The other conda-build tests that build packages, wind up calling get_build index and indirectly test the indexing code. Standalone conda-index has its own faster version of these tests that we delete here.